test/integration/test_utils.js

```diff
@@ -119,6 +119,17 @@ const waitForStorageEntries = async (page, nEntries) => {
 };
 exports.waitForStorageEntries = waitForStorageEntries;
 
+const waitForSerialized = async (page, nEntries) => {
+  await page.waitForFunction(
+    n =>
+      (window.PDFViewerApplication.pdfDocument.annotationStorage.serializable
+        ?.map?.size ?? 0) === n,
+    {},
+    nEntries
+  );
+};
+exports.waitForSerialized = waitForSerialized;
+
```

Comment:
I'm probably missing some details here but do we actually "need" this new function, or can we perhaps make use the existing `waitForStorageEntries` one just above instead?

Reply:
The annotationStorage can contain pre-existing annotations but if they haven't been changed they won't be serialized.